index.js

```diff
@@ -277,6 +277,81 @@ var MysqlDriver = Base.extend({
     }).nodeify(callback);
   },
 
+  createTable: function(tableName, options, callback) {
+    log.verbose('creating table:', tableName);
+    var columnSpecs = options;
+    var tableOptions = {};
+
+    if (options.columns !== undefined) {
+      columnSpecs = options.columns;
+      delete options.columns;
+      tableOptions = options;
+    }
+
+    var ifNotExistsSql = "";
+    if(tableOptions.ifNotExists) {
+      ifNotExistsSql = "IF NOT EXISTS";
+    }
+
+    var engineSql = '';
```

Comment:
May be should move that alltogether in one string, or do you see any reason to keep it like that. Would not like to have a huge list of parameters afterwards of strings which have no real reason to be sperated though :)